src/udev/udevadm-wait.c

```diff
@@ -367,6 +428,10 @@ int wait_main(int argc, char *argv[], void *userdata) {
         if (r < 0)
                 return log_error_errno(r, "Failed to set up device monitor: %m");
 
+        r = setup_periodic_timer(event);
+        if (r < 0)
+                return log_error_errno(r, "Failed to set up periodic timer: %m");
```

Comment:
grmbl, this seesm very hacky. I'd be really curious to understand why these uevents go missing. There's some bug we should try to track down instead of taping over.